.github/workflows/Check_CSV.yml

```diff
@@ -0,0 +1,25 @@
+# S. https://github.com/marketplace/actions/csvlinter for details
+# CAUTION: does not catch ALL csv errors
+# Better linter: https://csvlint.io
+name: Check if CSV files are valid
+
+on: 
+  [push, pull_request]
+
+permissions:
+  contents: read
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    name: csv lint
+    steps:
+    - uses: actions/checkout@v4
+    - name: get_files
+      run: |
+        csv_files=$(find -type f -ipath ".*csv" -printf "%p ")
```

Comment:
This returns (recursively) all csv files in the repo **case insensitive** (so **xy.csv**, **xy.CSV** or **xy.cSv** would be checked)